### PR TITLE
Deactivate Modeler by default, since it is deprecated

### PR DESCRIPTION
**Pull request by tgaugry**
______________________________________________________

This PR: 
- [ ] builds with SUCCESS for all platforms on the CI.
- [ ] does not generate new warnings.
- [ ] does not generate new unit test failures.
- [ ] does not generate new scene test failures.
- [ ] does not break API compatibility.
- [ ] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

File: applications/projects/CMakeLists.txt [hunk 1]
```diff
@@ -7,9 +7,7 @@ if(NOT Qt5Core_FOUND)
     set(SOFA_HAVE_QT_LIBRARIES 0)
 endif()
 
-if(SOFA_HAVE_QT_LIBRARIES)
-    sofa_add_application(Modeler Modeler OFF)
-endif()
```
Comment (guparan) [inline applications/projects/CMakeLists.txt:12]: Why? I would have kept the condition.

Comment (tgaugry) [reply applications/projects/CMakeLists.txt:12]: I did too at first, but it's a bit weird to do a check required for activation to then deactivate the thing. If the modeler require qt, then it should check for qt itself.

Comment (guparan) [reply applications/projects/CMakeLists.txt:12]: > If the modeler require qt, then it should check for qt itself.

I agree with that but you can't just remove the condition then, you should move it to Modeler's CMakeLists (and fail or warn+return if false).

Comment (tgaugry) [reply applications/projects/CMakeLists.txt:12]: Modeler does `find_package(Qt5 COMPONENTS Core QUIET)`
Maybe just change that to `REQUIRED` ?

Comment (guparan) [reply applications/projects/CMakeLists.txt:12]: I'm good with that :+1:

**Pull request merged**